portal/public/index.html

```diff
@@ -145,20 +145,38 @@
     </style>
 </head>
 <body>
-
-    <div class="mobile">
-        <div class="mobile-content">
-          <img src="/images/surflogo@3x.png">
-          <p>Het zoekportaal is gedurende de pilot niet te gebruiken op kleine schermen. Probeer je device te draaien of gebruik een computer.</p>
-          <p>During the pilot the search portal can't be used on small screens. Try to rotate your device or use a desktop computer.</p>
-        </div>
-    </div>
-
-    <div id="__nuxt">
-        <div id="nuxt-loading" aria-live="polite" role="status">
-            <div>Loading...</div>
-        </div>
-    </div>
-
+    <noscript>
+     <strong>Het zoekportaal heeft JavaScript nodig om te functioneren. Zet JavaScript aan in de browser.</strong>
+    </noscript>
+
+    <script type="text/javascript">
+        const isIE = /*@cc_on!@*/false || !!document.documentMode;
```

Comment:
Had even opgezocht, maar ik lijk te lezen dat javascript minifiers dit weghalen waardoor het niet meer werkt. Zie: http://www.pinlady.net/PluginDetect/IE/. Website ziet er wel uit als 1998, maar goed..

Reply:
Javascript minifiers runnen op dit moment niet meer. Ik heb wel moeite met const, want IE10 ondersteunt het niet. Ik weet dat de ticket over IE11 gaat, maar oudere browsers zou ook leuk zijn om mee te nemen met deze code.

Reply:
ok

Reply:
@fako , op productie is de JS wel geminified. Ik verwacht ook wel dat die build step dat doet en anders lijkt het mij een heel goed idee om dat op de korte termijn wel te gaan doen. Het enige verschil is ook dat we er een functie van moeten maken en meteen uitvoeren.